variables.tf

```diff
@@ -88,7 +88,11 @@ variable "cluster_description" {
   type        = string
   default     = "Generated by Terraform"
 }
-
+variable "cluster_labels" {
+  description = "The GCE resource labels (a map of key/value pairs) to be applied to the cluster. Both the key and the value must only contain lowercase letters ([a-z]), numeric characters ([0-9]), underscores (_) and dashes (-). International characters are allowed."
+  type        = map(string)
+  default     = {}
+}
```

Comment:
Let's add new empty lines before and after the variable declaration block

Reply:
done